api/openapi.yaml

```diff
@@ -1069,6 +1069,11 @@ components:
             - "FR3.3"
             - "FR3.3/GB/G2"
             - "GLOTT"
+        tags:
```

Comment:
Tags could be confusing with labels. What about a more clear name such as `categories`?